services/dy-static-file-server/src/dy_static_file_server/inputs_to_outputs.py

```diff
@@ -10,6 +9,12 @@
 from watchdog.events import DirModifiedEvent, FileSystemEventHandler
 from watchdog.observers import Observer
 
+# at runtime dy_static_file_server is not a module
+try:
```

Comment:
I would have expected that if it is not a module, the relative path would be the one failing ... could you double check this? :-)

Reply:
I'll rephrase that. The containing directory is not a module. At runtime relative imports do not work.